### ISSUE TITLE
HMXORDER gives incorrect output

### ISSUE THREAD

**Issue by ianhbell** [usnistgov/REFPROP-issues#266]
See #265; both orders of components (``A*B`` and ``B*A``) yield ``iFlag==1``, but only one should.  It might be related to binary pairs for which estimation of interaction parameters is active; that seems to be the only condition in common among the failures.

```python
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary
root = 'D:/Program Files (x86)/REFPROP'
RP = REFPROPFunctionLibrary(root)
RP.SETPATHdll(root)
print(RP.RPVersion())
fld0 = 'R23'
fld1 = 'R1234YF'
for fldA, fldB in [(fld0, fld1), (fld1, fld0)]:
    setfl = RP.SETFLUIDSdll(fldA + '*' + fldB)
    print(RP.HMXORDERdll(1, 2))
```
yields
```
10.0072
HMXORDERdlloutput(hcasi='20e26c20', hcasj='40377b40', iFlag=1, ierr=0, herr='')
HMXORDERdlloutput(hcasi='40377b40', hcasj='20e26c20', iFlag=1, ierr=0, herr='')
```

**Comment by EricLemmon** [comment]
I believe this one was fixed sometime between 10.0.0.72 and the current version.  I just tried the following code and obtained iFlag=1 on the first and fourth calls to HMXORDER and iFlag=0 on the other two.  On the last two, I called the FLAGS routine to force the code  to reread the HMX.BNC file in case that had anything to do with the problem.

```
      call SETFLUIDS ('H2S;PROPANE',i)
      call HMXORDER (1,2,hIDi,hIDj,iFlag,ierr,herr)

      call SETFLUIDS ('PROPANE;H2S',i)
      call HMXORDER (1,2,hIDi,hIDj,iFlag,ierr,herr)

      call FLAGS ('HMX',1,j,ierr,herr)
      call SETFLUIDS ('PROPANE;H2S',i)
      call HMXORDER (1,2,hIDi,hIDj,iFlag,ierr,herr)

      call FLAGS ('HMX',1,j,ierr,herr)
      call SETFLUIDS ('H2S;PROPANE',i)
      call HMXORDER (1,2,hIDi,hIDj,iFlag,ierr,herr)
```

**Comment by ianhbell** [comment]
Please confirm that exactly my calls yield the correct results.

**Comment by EricLemmon** [comment]
Interesting.  Didn't see that coming.  With your inputs of R23 and  R1234yf, you are correct that iFlag comes back as 1 in both cases.  But that is only because they are not in the HMX.BNC file and the order is not important since only the gammas are used.  In this case, I think the best thing to do is to send back a warning number and a message to this effect.  What are your thoughts?

**Comment by ianhbell** [comment]
But the order *is* important in general.  What about alkanes where the alkane+alkane model from the JCED paper is used for beta? Can this just be fixed so that the function works correctly?

**Comment by ianhbell** [comment]
I should clarify that I need this to work properly so that I can set betaT and gammaT when fitting, and I need to know whether I need to provide betaT or 1/betaT as the argument to ``SETKTVdll``.

**Comment by ianhbell** [comment]
Fixed in 10.0.0.78:
```
10.0078
HMXORDERdlloutput(hcasi='20e26c20', hcasj='40377b40', iFlag=-1, ierr=0, herr='')
HMXORDERdlloutput(hcasi='40377b40', hcasj='20e26c20', iFlag=-2, ierr=0, herr='')
```
